randovania/games/samus_returns/presets/multiworld-starter-preset.rdvpreset

```diff
@@ -0,0 +1,285 @@
+                        1
+                    ],
+                    "pickup_count": 30,
+                    "requires_main_item": true
+                },
+                "Power Bomb Tank": {
+                    "ammo_count": [
+                        1
+                    ],
+                    "pickup_count": 15,
+                    "requires_main_item": true
+                },
+                "Aeion Tank": {
+                    "ammo_count": [
+                        50
+                    ],
+                    "pickup_count": 15,
+                    "requires_main_item": true
+                }
+            }
+        },
+        "damage_strictness": 1.5,
+        "pickup_model_style": "all-visible",
+        "pickup_model_data_source": "etm",
+        "logical_resource_action": "randomly",
+        "first_progression_must_be_local": false,
+        "minimum_available_locations_for_hint_placement": 5,
+        "minimum_location_weight_for_hint_placement": 0.2,
+        "dock_rando": {
+            "mode": "vanilla",
+            "types_state": {
+                "door": {
+                    "can_change_from": [
+                        "Charge Beam Door",
+                        "Missile Door",
+                        "Plasma Beam Door",
+                        "Power Beam Door",
+                        "Power Bomb Door",
+                        "Spazer Beam Door",
+                        "Super Missile Door",
+                        "Wave Beam Door"
+                    ],
+                    "can_change_to": [
+                        "Charge Beam Door",
+                        "Missile Door",
+                        "Plasma Beam Door",
+                        "Power Beam Door",
+                        "Power Bomb Door",
+                        "Spazer Beam Door",
+                        "Super Missile Door",
+                        "Wave Beam Door"
+                    ]
+                }
+            }
+        },
+        "single_set_for_pickups_that_solve": true,
+        "staggered_multi_pickup_placement": true,
+        "check_if_beatable_after_base_patches": false,
+        "teleporters": {
+            "mode": "vanilla",
+            "excluded_teleporters": [],
+            "excluded_targets": []
+        },
+        "energy_per_tank": 100,
+        "starting_energy": 99,
+        "starting_aeion": 1000,
+        "life_tank_size": 299,
+        "aeion_tank_size": 500,
+        "missile_tank_size": 30,
+        "super_missile_tank_size": 10,
+        "elevator_grapple_blocks": false,
+        "area3_interior_shortcut_no_grapple": true,
+        "charge_door_buff": true,
+        "beam_door_buff": true,
+        "beam_burst_buff": true,
+        "nerf_super_missiles": false,
+        "surface_crumbles": true,
+        "area1_crumbles": true,
+        "reverse_area8": false,
+        "allow_highly_dangerous_logic": false,
+        "artifacts": {
+            "prefer_metroids": true,
```

Comment:
if prefer anywhere is true, then prefer metroids and prefer_stronger_metroids should probably get disabled in the preset just in case:tm:

Reply:
Isn't that just how we toggle the stuff and gave them priority?  I mean, it's doable but that would not be the file how it will be created when you do it via the gui.

Reply:
not as far as i can see. the GUI does this right now:
```py
    def _on_free_placement(self, value: bool) -> None:
        if not value:
            return
        self.prefer_metroids_check.setEnabled(False)
        self.prefer_stronger_metroids_check.setEnabled(False)
        self.prefer_bosses_check.setEnabled(False)
```

Reply:
Ok, then we need to ask @dyceron

Reply:
I created the preset within rdv. Nothing was manually edited.

Reply:
huh ok then. maybe the function just doesnt trigger if the checkbox is manually updated or something.